code/game/objects/items/stacks/sheets/glass.dm

```diff
@@ -123,9 +123,11 @@ GLOBAL_LIST_INIT(pglass_recipes, list ( \
  */
 GLOBAL_LIST_INIT(reinforced_glass_recipes, list ( \
 	new/datum/stack_recipe("windoor frame", /obj/structure/windoor_assembly, 5, time = 0, on_floor = TRUE, window_checks = TRUE), \
-	null, \
 	new/datum/stack_recipe("directional reinforced window", /obj/structure/window/reinforced/unanchored, time = 0, on_floor = TRUE, window_checks = TRUE), \
-	new/datum/stack_recipe("fulltile reinforced window", /obj/structure/window/reinforced/fulltile/unanchored, 2, time = 0, on_floor = TRUE, window_checks = TRUE) \
+	//austation begin -- glass floors
+	new/datum/stack_recipe("fulltile reinforced window", /obj/structure/window/reinforced/fulltile/unanchored, 2, time = 0, on_floor = TRUE, window_checks = TRUE), \
```

Comment:
What’s this?

Reply:
Just added a "," for it to work

Reply:
As in. What’s this about a full tile r-window.

Reply:
It's the stack recipe for crafting the fulltile window when you click it inhand

Reply:
I extended the comment to there because a comma is added

Reply:
Don’t we already have that recipe though :p

Reply:
Ah alright